AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -67,35 +73,34 @@ export var functionsFlow = {
             var propertyName = "AzureWebJobsStorage";
             // Using anchor tag instead of markdown link as we need the link to open in a new window/tab instead of the current iFrame which is disallowed
             var failureDetailsMarkdown = `Please refer to <a href= "https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings#azurewebjobsstorage" target="_blank">this documentation</a> on how to configure the app setting "${propertyName}".`;
-            var connectionString = appSettings[propertyName];
-            if (connectionString != undefined) {
+            var connectionString = undefined;
+            if(isDaasNew){
+                var connectionStringType = isDaasExtAccessible ? ConnectionStringType.BlobStorageAccount : undefined;
+            }else{
                 var connectionStringType = isDaasExtAccessible ? ConnectionStringType.StorageAccount : undefined;
+            }                
                 var subChecksL2 = await networkCheckConnectionString(propertyName, connectionString, connectionStringType, dnsServers, diagProvider, isVnetIntegrated, failureDetailsMarkdown);
                 var maxCheckLevel = getMaxCheckLevel(subChecksL2);
                 var title = maxCheckLevel == 0 ? `Network connectivity test to Azure storage endpoint configured in app setting "${propertyName}" was successful.` :
                     `Network connectivity test to Azure storage endpoint configured in app setting "${propertyName}" failed.`;
                 subChecksL1.push({ title: title, subChecks: subChecksL2, level: maxCheckLevel });
-            } else {
-                subChecksL1.push({
-                    title: `The app setting "${propertyName}" is not configured.  The Function App cannot work without this essential setting.`,
-                    level: 2,
-                    detailsMarkdown: failureDetailsMarkdown
-                });
-            }
-
+           
             // WEBSITE_CONTENTAZUREFILECONNECTIONSTRING
             propertyName = "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING";
             failureDetailsMarkdown = `Please refer to <a href= "https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings#website_contentazurefileconnectionstring" target="_blank">this documentation</a> on how to configure the app setting "${propertyName}".`;
-            connectionString = appSettings[propertyName];
-            if (connectionString != undefined) {

```

Comment:
This effectively makes the optional app setting required.  I am unable to determine if this app setting can be set via managed identity and if so, what the postfix is.  Either way, we need to check for both the app setting and the postfix variant here to avoid reporting an error when the setting is not defined.